packages/web-components/src/anchored-region/index.ts

```diff
@@ -0,0 +1,25 @@
+import { customElement } from '@microsoft/fast-element';
+import { AnchoredRegion, AnchoredRegionTemplate as template } from '@microsoft/fast-foundation';
+import { AnchoredRegionStyles as styles } from './anchored-region.styles';
+
+/**
+ * The FAST AnchoredRegion Element. Implements {@link @microsoft/fast-foundation#AnchoredRegion},
+ * {@link @microsoft/fast-foundation#AnchoredRegionTemplate}
+ *
+ *
+ * @beta
+ * @remarks
+ * HTML Element: \<fluent-anchored-region\>
+ */
+@customElement({
+  name: 'fluent-anchored-region',
+  template,
+  styles,
```

Comment:
Do we want the shadow DOM closed for this?

Reply:
I am unsure if the fluentui version should be closed, but chatting with Stephane he suggested it opened, so I kept it the same here.

Reply:
Currently all other Fluent UI controls are being exported as "closed" (https://github.com/microsoft/fluentui/issues/15346) and as such, this should follow the current implementation.